Master

adamdjbrett opened the pull request:
# Conflicts:
#	README.md
#	index.md

blmoore commented:
Did you mean to push your cv back to my repo — are there some generic improvements worth cherry-picking?

Also the json-ld is neat, did you write it by hand?

adamdjbrett commented:
Benjamin,

I'm so sorry for screwing up your repo. I was trying to update my CSS with
the improvements that you had made to your CV. I'm so sorry for messing it
up.

The biggest differences are:

   1. I hand wrote JSON-LD to add to the template to help with SEO
   2. added fav icons
   3. added facebook open graph
   4. minified and combined the font CSS so there is one less css call
   5. localized the fonts to help speed things up.
   6. I also changed the colors to match Syracuse University new color
   scheme so my orange is now #D44500

With these changes I score in the 89-97% range on GMetrix
https://gtmetrix.com/reports/cv.adamdjbrett.com/K3GZnNjQ

Over the summer I plan on working to combine and minify, cv-screen.css,
combo.css and doing some tweaking to make the template fully responsive
(hopefully)

I want to say thank you so much for such a wonderful markdown CV template.
I've looked at quite literally all the markdown CV options and yours is the
most simplified, direct, and aesthetically pleasing. Thank you for all the
work that you have put into this project.



Regards,

Adam DJ Brett

On Wed, May 3, 2017 at 4:29 AM, Benjamin L. Moore <notifications@github.com>
wrote:

> Did you mean to push your cv back to my repo — are there some generic
> improvements worth cherry-picking?
>
> Also the json-ld is neat, did you write it by hand?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/blmoore/md-cv/pull/3#issuecomment-298851372>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AVnPQh0WIrth9GIQovdu344HA8y08j34ks5r2Dr9gaJpZM4NMn3n>
> .
>
